library/src/hiptensor.cpp

```diff
@@ -151,8 +151,7 @@ hiptensorStatus_t hiptensorInitTensorDescriptor(const hiptensorHandle_t*     han
         return HIPTENSOR_STATUS_NOT_INITIALIZED;
     }
 
-    if((lens == nullptr)
-       || ((dataType != HIP_R_16F) && (dataType != HIP_R_32F) && (dataType != HIP_R_64F))
+    if((lens == nullptr) || ((dataType != HIP_R_32F) && (dataType != HIP_R_64F) && (dataType != HIP_C_32F) && (dataType != HIP_C_64F))
```

Comment:
It's nice to support HIP_C_32F and HIP_C64F. And could we keep the support to HIP_R_16F as well since permute has been supporting f16. Permute_test cannot pass without it.